ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/TheirDetailsMapper.java

```diff
@@ -100,10 +103,19 @@ private TheirDetails getCompany(CCDParty ccdParty) {
 
     private TheirDetails getIndividual(CCDParty ccdParty) {
         CCDIndividual ccdIndividual = ccdParty.getIndividual();
-        return new IndividualDetails(ccdIndividual.getName(), addressMapper.from(ccdIndividual.getAddress()),
-            ccdIndividual.getEmail(), representativeMapper.from(ccdIndividual.getRepresentative()),
-            addressMapper.from(ccdIndividual.getCorrespondenceAddress()),
-            LocalDate.parse(ccdIndividual.getDateOfBirth(), ISO_DATE));
+        return new IndividualDetails(ccdIndividual.getName(),
+            addressMapper.from(ccdIndividual.getAddress()),
+            ccdIndividual.getEmail(),
+            Optional.ofNullable(ccdIndividual.getRepresentative()).isPresent()
+                ? representativeMapper.from(ccdIndividual.getRepresentative())
+                : null,
+            Optional.ofNullable(ccdIndividual.getCorrespondenceAddress()).isPresent()
+                ? addressMapper.from(ccdIndividual.getCorrespondenceAddress())
+                : null,
+            Optional.ofNullable(ccdIndividual.getDateOfBirth()).isPresent()
```

Comment:
I think using just an `if` statement would suffice. Wrapping in `Optional` unnecessarily creates extra objects in memory...